.github/workflows/starknet-js-tests.yml

```diff
@@ -0,0 +1,37 @@
+name: Run Tests
+
+on:
+  workflow_dispatch:
+  workflow_call:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    env:
+      BINARY_PATH: ../target/release/madara
+    steps:
+      - uses: actions/checkout@v3
+      - uses: Swatinem/rust-cache@v2
+        with:
+          shared-key: "cache"
+          save-if: false
+      - uses: actions/cache@v3
+        with:
+          path: target/release/madara
+          key:
+            ${{ runner.os }}-cargo-${{ hashFiles('Cargo.lock') }}-${{
+            github.run_id }}
+          fail-on-cache-miss: true
+      - name: Setup dev chain
+        run: |
+          ./target/release/madara setup --chain=dev --from-remote
+      - name: Run starknet-js test
+        run: |-
+          ./target/release/madara --dev --tmp --rpc-external --execution native --pool-limit=100000 --pool-kbytes=500000 --rpc-methods=unsafe --rpc-cors=all --in-peers=0 --out-peers=1 --no-telemetry &
+          NATIVE_RUN_PID=$!
+          while ! echo exit | nc localhost 9944; do sleep 1; done
+          git clone https://github.com/petscheit/madara-js-tests.git
```

Comment:
It's more than just madara, it should be every sequencer.
`sequencer-starknet-js-tests`

Reply:
we can rename when I transfer the repo